ui/page/settings/app_settings_page.go

```diff
@@ -647,6 +680,83 @@ func (pg *AppSettingsPage) HandleUserInteractions() {
 			pg.ParentWindow().ShowModal(currentPasswordModal)
 		}
 	}
+
+	if pg.backupDEX.Clicked() {
+		// Show modal asking for dex password and then reveal the seeds.
+		dexPasswordModal := modal.NewCreatePasswordModal(pg.Load).
+			EnableName(false).
+			EnableConfirmPassword(false).
+			Title(values.String(values.StrDexPassword)).
+			SetPositiveButtonCallback(func(_, password string, pm *modal.CreatePasswordModal) bool {
+				var err error
+				defer func() {
+					if err != nil {
+						pm.SetError(err.Error())
+						pm.SetLoading(false)
```

Comment:
Don't see pm.SetLoading(true) anywhere here.

Reply:
It's set by the password modal before calling the callback: https://vscode.dev/github/crypto-power/cryptopower/blob/backup-dex/ui/modal/create_password_modal.go#L238